applications/com.github.sammarxz.pomerode.json

```diff
@@ -0,0 +1,5 @@
+{
+  "source": "https://github.com/sammarxz/pomerode.git",
+  "commit": "6ef1f911d2d69d3bbd72e5c42b6e674c09eee085",
+  "version": "v1.0.0"
```

Comment:
You created a tag named [v1.0](https://github.com/sammarxz/pomerode/tree/v1.0), not `v1.0.0`. You need to create `v1.0.0` tag on your repository